src/src_V2/modules/auth/login/Login.jsx

```diff
@@ -61,7 +64,7 @@ export default function Login() {
           LOGIN
         </Button>
         <Typography>
-          {'Don\'t have an account? '}
+          {"Don't have an account? "}
```

Comment:
pls remove curly brackets.

Reply:
it is done to keep the space with string